docs/sources/operations/parquet/schema.md

```diff
@@ -0,0 +1,274 @@
+* Trace ID - Don't store on each span.
+* Root service/span names/StartTimeUnixNano - These are selected properties of the root span in each trace (if there is one). These are used for displaying results in the Grafana UI. These properties are computed at ingest time and stored once for efficiency, so we don't have to find the root span.
+* `DurationNanos` - The total trace duration, computed at ingest time. This powers the min/max duration filtering in the current Tempo search and is more efficient than scanning the spans duration column. However, it may go away with TraceQL or we could decide to change it to span-level duration filtering too.
+
+## Dedicated columns
+
+Projecting attributes to their own columns has benefits for speed and size.
+Therefore we are taking an opinionated approach and projecting some common attributes to their own columns.
+All other attributes are stored in the generic key/value maps and are still searchable, but not as quickly.
+We chose these attributes based on what we commonly use ourselves (scratching our own itch), but we think they will be useful to most workloads.
+
+Resource-level attributes include the following:
+
+* `service.name`
+* `cluster` and `k8s.cluster.name`
+* `namespace` and `k8s.namespace.name`
+* `pod` and `k8s.pod.name`
+* `container` and `k8s.container.name`
+
+Span-level attributes include the following:
+
+* `http.method`
+* `http.url`
+* `http.status_code` (int)
+
+## "Any"-type Attributes
+
+OTLP attributes have variable data types, which is easy to accomplish in formats like protocol-buffers, but does not translate directly to Parquet.
+Each column must have a concrete type.
+There are several possibilities here but we chose to have optional values for each concrete type.
+`Array` and `KeyValueList` types are stored as protocol-buffer-encoded byte arrays.
+
+```yaml
+repeated group Attrs {
+    required binary  Key (STRING);
+
+    # Only one of these will be set
+    optional binary  Value (STRING);
+    optional boolean ValueBool;
+    optional double  ValueDouble;
+    optional int64   ValueInt (INT(64,true));
+    optional binary  ValueArray (STRING);
+    optional binary  ValueKVList (STRING);
+}
+```
+
+Pros:
+
+* Preserves the functionality of parquet column min/max statistics which can be leveraged during search
+* Native data types should be more efficient to store and process
+
+Cons:
+
+* Many nulls - as each attribute only populates 1 column, the other 5 are guaranteed to be null. This has a non-trivial impact on storage size.
+
+## Event attributes
+
+Span event attributes are stored as JSON-encoded strings in a generic key/value map. This is by far the most space-efficient encoding and the trade-off of decreased searchability seems worthwhile. Storing event attributes this way reduces the block size ~16% for our dataset, which is huge. There are currently no use cases to search event attributes, but we can revisit this in the future if needed.
```

Comment:
This is outdated and event attributes are stored as protocol buffer encoded.  We can also remove the historical discussion of pros/cons.

Reply:
I've updated based upon what you said by removing the entire paragraph and replacing it with a statement that "Event attributes are stored as protocol-buffered encoded." 

I didn't remove the example, but based upon what you said, maybe it should be removed?